conn.go

```diff
@@ -643,13 +642,12 @@ func (c *Conn) handleQueuedPackets(ctx context.Context) error {
 			}
 		}
 		var e *alertError
-		if errors.As(err, &e) {
-			if e.IsFatalOrCloseNotify() {
-				return e
-			}
-		} else if err != nil {
+		if errors.As(err, &e) && e.IsFatalOrCloseNotify() {
 			return e
 		}
+		if err != nil {
```

Comment:
I'm not entirely sure this makes sense? We're first checking if it's a particular error with a particular condition and then `return e`, but that's still of type `error`. Right after, we're doing `if err != nil { return err }`. It seems at that point we don't need the check for `if errors.As` at all, since we always return `err` if it's not nil.

Reply:
I think you're right. I copied this from master. 

I think all we need is `if err != nil {return err}`

Reply:
Thanks for fixing that. I think you might've forgotten to push the right branch? I don't see the change and CI hasn't picked up on it.

Reply:
done.

Reply:
Adding the `else if err != nil` in place of `if err != nil` to keep the current behavior as is.